### ISSUE TITLE
Setting `icmp_check` on an imported `statuscake_uptime_check` resource force replacement

### ISSUE THREAD

**Issue by weaversam8** [StatusCakeDev/terraform-provider-statuscake#104]
**Describe the bug**
We're attempting to bring our existing StatusCake configuration into Terraform. I start, I created configuration that I thought matched our existing resource, and then `terraform import`ed the resource with the ID from the StatusCake console. When I run a `terraform plan` though, the plan forces replacement of my resource, based on the fact that the `icmp_check` block is provided, even though the check I imported was an ICMP check.

When I run a `terraform state show` on the imported resource, _none_ of the `dns_check,http_check,icmp_check,tcp_check` blocks appear.

**To Reproduce**
1. Create an ICMP check via the StatusCake UI
2. Create terraform configuration for that `statuscake_uptime_check`
3. Use `terraform import` to import the existing check into the state
4. Try a `terraform plan`
5. The resource should be marked for replacement based on the ICMP check block.

**Expected behavior**
I expected the resource to only require a few modifications, not complete replacement.

**Screenshots**
<img width="309" alt="image" src="https://user-images.githubusercontent.com/2546219/206030112-738525b8-3f94-4490-815f-20ca9999b885.png">

**Comment by tomasbasham** [comment]
Thanks for reporting this. Can you post the corresponding HCL for this test.

**Comment by weaversam8** [comment]
I can't post the exact HCL, as it's from an internal project, but here's a redacted analogue.

```
resource "statuscake_uptime_check" "statuscake" {
  for_each = {
    for idx, cname in aws_route53_record.dns_name : idx => cname
  }

  name = "${aws_instance.instance[each.key].tags["Name"]}-ping"
  check_interval = 60
  confirmation = 3
  trigger_rate = 1

  contact_groups = [ "12345" ]

  monitored_resource {
    address = aws_route53_record.dns_name[each.key].name
  }

  regions = [
    "new-york",
    "georgia",
    "illinois",
    "florida",
    "texas",
    "california"
  ]

  icmp_check {
    enabled = true
  }
}
```

**Comment by tomasbasham** [comment]
I've found that the provider is ignoring the ICMP test type when it is read from the API. This will be fixed in a release shortly.

**Comment by tomasbasham** [comment]
This has been merged into master for release

**Comment by tomasbasham** [comment]
This has been released in version 2.0.6
